test/runtests.jl

```diff
@@ -22,6 +22,7 @@ base = ["test_abstract_vectors.jl",
         "test_composite.jl",
         "test_judiWeights.jl",
         "test_judiWavefield.jl",
+        "test_judiVStack.jl",
```

Comment:
there is already a test for it , `test_composite.jl`

Reply:
Oh I could rename it as test_joCoreBlock.jl. In "test_composite.jl", we only test the linearity and stuff but the new test I wrote is to test the forward and adjoint of the joCoreBlock operator on judiVStack structure.

Reply:
No, just look at how the test are setup and add the extra one. 

>test I wrote is to test the forward and adjoint of the joCoreBlock operator

joCoreBlock are JOLI objects and already tested in JOLI including adjoint. You are testing the part you are adding wich is the multiplication. Just build a joCorBLock, do the multiplication and test you get what you expect.
 `joOnes` is tested in `judWeights` for example